docs/guide/concept-di-container.md

```diff
@@ -224,11 +224,11 @@ and the container will automatically resolve dependencies by instantiating them
 them into the newly created objects. The dependency resolution is recursive, meaning that
 if a dependency has other dependencies, those dependencies will also be resolved automatically.
 
-You can use [[yii\di\Container::get()]] to create new objects. The method takes a dependency name,
+You can use [[yii\di\Container::get()|get()]] to create new objects. The method takes a dependency name,
 which can be a class name, an interface name or an alias name. The dependency name may or may
-not be registered via `set()` or `setSingleton()`. You may optionally provide a list of class
-constructor parameters and a [configuration](concept-configurations.md) to configure the newly created object.
-For example,
+not be registered via [[yii\di\Container::set()|set()]] or [[yii\di\Container::setSingleton()|setSingleton()]].
```

Comment:
The dependency name may or may not be registered -> The dependency name may be registered
("may" already implies "maybe not" ;) )